tophat/yvm
# chore: install messaging around invalid version tags

## Description

- Updated the install script to query github to get version tags for validation

### Checklist
- [x] This PR has updated documentation
- [x] This PR has sufficient testing

## DevQA

### DevQA Steps

1. Install script latest version
```sh
curl -fsSL https://raw.githubusercontent.com/tophat/yvm/chore-343/scripts/install.js | node
```
- Launch new shell
- `yvm version` - should be latest version

2. Install script loosely specified version
```sh
curl -fsSL https://raw.githubusercontent.com/tophat/yvm/chore-343/scripts/install.js | INSTALL_VERSION=2.4 node
```
- Launch new shell
- `yvm version` - should be version satisfying install version

3. Install script strictly specified version
```sh
curl -fsSL https://raw.githubusercontent.com/tophat/yvm/chore-343/scripts/install.js | INSTALL_VERSION=v3.0.0 node
```
- Launch new shell
- `yvm version` - should be version satisfying install version

### Comments

- Closes #343
